### ISSUE TITLE
Taskbard icon is pixelated on 64bit Windows

### ISSUE THREAD

**Issue by jakezatecky** [keeweb/keeweb#1343]
**Describe the bug**
This report is related to #531 and #843.

On the 64bit version of Windows, the KeeWeb icon remains pixelated in the taskbar:

![image](https://user-images.githubusercontent.com/3210703/68445176-25966180-01a7-11ea-8359-182296e47fb3.png)
![image](https://user-images.githubusercontent.com/3210703/68445239-56769680-01a7-11ea-8831-385c74e7b184.png)

The previous PR seems to have fixed this for 32bit Windows, as the [PR specifically checks against 32bit Windows](https://github.com/keeweb/keeweb/pull/843/files#diff-7058cf17637d9a2b6b8c5044f4fc14e4R149). I was tempted to create a PR that also checks `process.platform !== 'win64'` or that removes the `icon` property altogether (seeing as that is what Atom did), but I could not build the application without error.

**To Reproduce**
1. Download and install 64bit version of KeeWeb on Windows 10
2. Launch application and look at the taskbar icon

**Expected behavior**
Taskbar icon is sharp and clean.

**Environment**
KeeWeb v1.12.3 (9b07bbd5, 2019-11-06)
Environment: electron v7.0.0
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.12.3 Chrome/78.0.3905.1 Electron/7.0.0 Safari/537.36

**Kdbx File**
N/A

**Logs**
```
2019-11-08T02:01:30.569Z [app] Started in 830ms ¯\_(ツ)_/¯
2019-11-08T02:01:30.571Z [updater] Next update check will happen in 86307s
```

**Comment by antelle** [comment]
Hi! There's no such thing as `win64` platform, it's always `win32` for historical reasons, so it needs more thinking.

**Comment by jakezatecky** [comment]
Ahh, you're right it does return `win32` regardless of architecture (`process.arch` returns `ia32` vs. `x64`). My mistake!

I am at a loss for what could be causing this.

**Comment by Sigmanor** [comment]
Same here
![18V7xS8](https://user-images.githubusercontent.com/7986067/68855745-97126a80-06e7-11ea-9602-bceae0314dae.png)

Icon in tray seems pixelated too
![](https://i.imgur.com/SWkyokX.png)

**Environment**
KeeWeb v1.12.3 (9b07bbd5, 2019-11-06)
Environment: electron v7.0.0
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.12.3 Chrome/78.0.3905.1 Electron/7.0.0 Safari/537.36

**Comment by lttr** [comment]
Also on my Linux setup:
![Screenshot from 2020-01-22 10-57-54](https://user-images.githubusercontent.com/4193121/72884527-9230ef00-3d06-11ea-9638-723f54045ae0.png)

**Environment**
KeeWeb v1.12.3 (9b07bbd5, 2019-11-06)

Pop!_OS 19.04 with Gnome extension Dash to Panel

**Comment by jakezatecky** [comment]
For Windows I managed to somehow fix the issue by searching for KeeWeb in the Start Menu, right-clicking it, unpinning it from the Start Menu, pinning it to the taskbar, unpinning it from the taskbar, and then launching KeeWeb again. The icon issue went away.

I think this is related to some bizarre icon caching issues with Windows. At some point in the past, I had the old version of KeeWeb that had pixelation issues (#531). When the newer versions of KeeWeb installed, Windows never fixed the icon.

Deleting the `IconCache.db` files did not solve the issue. Only pinning/unpinning from the Start Menu search results fixed the issue for me.

I hope this helps someone else. I spent several hours today trying to fix icon-related issues on my Windows account. I came to the conclusion this was not KeeWeb's fault, but Window's fault.
